react-app/src/components/GridPage.tsx

```diff
@@ -0,0 +1,170 @@
+import { CSSProperties, useState } from "react";
+import styles from "../styles/GridPage.module.css";
+import { colors } from "../static/GridElements";
+import GridConfigButton from "./UI/GridButton";
+import GridButtonWindow from "./GridButtonWindow";
+import { GridConfigSlider } from "./GridConfigSlider.tsx";
+import { subtractWithSaturation8bit } from "../utils.tsx";
+import { CodeListing } from "./CodeListing.tsx";
+import Grid from "./Grid.tsx";
+
+export default function GridPage() {
+  const numberOfElements = 20;
+
+  const [cssProps, setCssProps] = useState<CSSProperties>({
+    gridAutoFlow: "dense",
+    gridTemplateColumns: "100",
+    gridAutoRows: "100",
+    gap: "10",
+  });
+
+  const [selectedAspects, setSelectedAspects] = useState([
+    { id: 1, aspect: "aspect_1_to_2", selected: true },
```

Comment:
aspect could be an enum

Reply:
yes, but then I will have to implement special mapper to css classes or do it other way